### PR TITLE
QuickStart: ZAPit first phase

### PR DESCRIPTION
**Pull request by psiinon**
## Overview
Briefly describe the purpose, goals, and changes or improvements made in this pull request.

## Related Issues
Specify any related issues or pull requests by linking to them.

## Checklist
- [x] Update help
- [x] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [ ] Write tests
- [ ] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

For more details, please refer to the [developer rules and guidelines](https://www.zaproxy.org/docs/developer/dev-rules-and-guidelines/).

### CODE REVIEW

File: addOns/quickstart/src/main/javahelp/org/zaproxy/zap/extension/quickstart/resources/help/contents/zapit.html
Comment (kingthorin) [inline addOns/quickstart/src/main/javahelp/org/zaproxy/zap/extension/quickstart/resources/help/contents/zapit.html:1]: Throughout the help ZAP IT, ZAP It, ZAPit, etc should be consistent.

**Review by kingthorin — COMMENTED** [review]
Seems like a good start

File: addOns/quickstart/src/main/javahelp/org/zaproxy/zap/extension/quickstart/resources/help/contents/zapit.html
Comment (psiinon) [reply addOns/quickstart/src/main/javahelp/org/zaproxy/zap/extension/quickstart/resources/help/contents/zapit.html:1]: Good point. Have standardised on `ZAPit` 😁

**Review by kingthorin — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**